src/viam/api/config/invoke_buf_generate.cmake.in

```diff
@@ -0,0 +1,7 @@
+set(ENV{PATH} "$ENV{PATH}:@PROTOC_BIN_PATH@")
```

Comment:
What is the scope of this modified environment?

Reply:
I think just the subshell which runs the script but let me make sure of that.

Reply:
As above, we don't actually need this file